### ISSUE TITLE
CORS is not enabled by default

### ISSUE THREAD

**Issue by juancarlosjr97** [directus/next#871]
Steps to reproduce:

1. Create a new instance of directus `npx create-directus-project testing-directus-locally` and then `npx directus start`
2. Execute the command

```
fetch('http://localhost:8055/graphql', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
    'Accept': 'application/json',
  },
  body: JSON.stringify({query: "query allSchemaTypes { __schema { types { name kind description } } }"})
})
  .then(r => r.json())
  .then(console.log);
```

3. The result is "rejected" and the expected result is "fulfilled"

4. Go to `.env` and add `CORS_ENABLED="true"`

5. Rerun `npx directus start`

6. Execute

```
fetch('http://localhost:8055/graphql', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
    'Accept': 'application/json',
  },
  body: JSON.stringify({query: "query allSchemaTypes { __schema { types { name kind description } } }"})
})
  .then(r => r.json())
  .then(console.log);
```

and the result will be `fulfilled` as it should be expected.

------

CORS should be enabled by default which is not enabled by default

**Comment by juancarlosjr97** [comment]
I just found out that is disabled by default, compared to the documentation that says is enabled

`node_modules/directus/dist/env.js` - `CORS_ENABLED: false,`

Either update documentation or enable CORS by default :)

**Comment by rijkvanzanten** [comment]
Fixed in #872
